README.md

```diff
@@ -36,15 +46,96 @@ Usage
     itk.imwrite(median, output_filename)
 ```
 
+### Other Resources for Using ITK in Python
+
 See also the [ITK Python Quick Start
 Guide](https://itkpythonpackage.readthedocs.io/en/master/Quick_start_guide.html).
-There are also many [downloadable examples documented in
-Sphinx](https://itk.org/ITKExamples/search.html?q=Python).
+There are also many [downloadable examples on the ITK examples website](https://examples.itk.org/search.html?q=Python).
 
 For more information on ITK's Python wrapping, [an introduction is
 provided in the ITK Software
 Guide](https://itk.org/ITKSoftwareGuide/html/Book1/ITKSoftwareGuide-Book1ch3.html#x32-420003.7).
 
+## Building with ITKPythonPackage
+
+ITK reusable workflows are available to build and package Python wheels as
+part of Continuous Integration (CI) via Github Actions runners.
+Those workflows can handle the overhead of fetching, configuring, and
+running ITKPythonPackage build scripts for most ITK external modules.
+See [ITKRemoteModuleBuildTestPackageAction](https://github.com/InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction)
+for more information.
+
+For special cases where ITK reusable workflows are not a good fit,
+ITKPythonPackage scripts can be directly used to build Python wheels
+to target Windows, Linux, and MacOS platforms. See
+[ITKPythonPackage ReadTheDocs](https://itkpythonpackage.readthedocs.io/en/master/Build_ITK_Module_Python_packages.html)
+documentation for more information on building wheels by hand.
+
+## Frequently Asked Questions
+
+### What target platforms and architectures are supported?
```

Comment:
@thewtex Could you please confirm whether this list is correct?

Reply:
@tbirdso 

*Ubuntu 20.04+ x86_64 platforms* -> *Linux glibc 2.17+ (E.g. Ubuntu 18.04+) x86_64 platforms*.

We also have Linux / ARM, correct? By fixing the Docker image version, I think external module wheels work, too?

Reply:
> Ubuntu 20.04+ x86_64 platforms -> Linux glibc 2.17+ (E.g. Ubuntu 18.04+) x86_64 platforms.

Thanks, will update.

> We also have Linux / ARM, correct? By fixing the Docker image version, I think external module wheels work, too?

It may be possible to build Linux ARM wheels in the right environment under the status quo, but there are a few fixes in https://github.com/InsightSoftwareConsortium/ITKPythonPackage/pull/236/files that in my opinion should be merged in before we list it as a "supported" option. The Docker image version fix will allow that PR to move forward with more testing, yes.

Reply:
Updated, for now I've listed Linux ARM as "coming soon".